susemanager/src/mgr-create-bootstrap-repo

```diff
@@ -489,6 +490,18 @@ def create_repo(label, options, mgr_bootstrap_data, additional=[]):
     log("The bootstrap repo should contain the following packages: {0}".format(packagelist), 2)
 
     debs_dir = os.path.join(destdirtmp, 'debs')
+    if repotype == 'deb' and not options.dryrun:
+        # reprepro is picky with packages having the same name, but different checksums.
+        # To avoid aborting, we copy the old packages to debs and add or overwrite them
+        # with new found packages and run reprepro again (bsc#1184330)
```

Comment:
I don't really understand the last part of the comment "run reprepro again". I think `reprepro` was not executed before getting to this point.

Reply:
Every reposync trigger this script and we operate on the already created repository from the run before.
For rpm with createrepo_c this is not a problem, but reprepro is picky and has conditions where it fail.
We hit such a condition in the bug.

This change is to workaround such problems.